examples/panda/panda_goal_reaching/controllers/robot_supervisor_manager/robot_supervisor_ddpg.py

```diff
@@ -0,0 +1,225 @@
+        Type: Box(10)
+        Num	Observation                Min(rad)      Max(rad)
+        0	Target x                   -Inf           Inf
+        1	Target y                   -Inf           Inf
+        2	Target z                   -Inf           Inf
+        3	Position Sensor on A1      -2.8972        2.8972
+        4	Position Sensor on A2      -1.7628        1.7628
+        5	Position Sensor on A3      -2.8972        2.8972
+        6	Position Sensor on A4      -3.0718       -0.0698
+        7	Position Sensor on A5      -2.8972        2.8972
+        8   Position Sensor on A6      -0.0175        3.7525
+        9	Position Sensor on A7      -2.8972        2.8972
+        
+    Actions:
+        Type: Continuous
+        Num	  Min   Max   Desc
+        0	  -1    +1    Set the motor position from θ to θ + (action 0)*0.032
+        ...
+        6     -1    +1    Set the motor position from θ to θ + (action 6)*0.032
+    Reward:
+        Reward is - 2-norm for every step taken (extra points for getting close enough to the target)
+    Starting State:
+        [Target x, Target y, Target z, 0, 0, 0, -0.0698, 0, 0, 0]
+    Episode Termination:
+        distance between "endEffector" and "TARGET" < 0.005 or reached step limit
+        Episode length is greater than 300
+        Solved Requirements (average episode score in last 100 episodes > -100.0)
+    """
+
+    def __init__(self):
+        """
+        In the constructor the observation_space and action_space are set and references to the various components
+        of the robot required are initialized here.
+        """
+
+        super().__init__()
+
+        # Set up gym spaces
+        self.observation_space = Box(low=np.array([-np.inf, -np.inf, -np.inf, -2.8972, -1.7628, -2.8972, -3.0718, -2.8972, -0.0175, -2.8972]),
+                                     high=np.array([np.inf,  np.inf,  np.inf, 2.8972,  1.7628,  2.8972, -0.0698,  2.8972,  3.7525,  2.8972]),
+                                     dtype=np.float64)
+        self.action_space = Box(low=np.array([-1.0, -1.0, -1.0, -1.0, -1.0, -1.0, -1.0]), high=np.array([1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0]), dtype=np.float64)
+
+        # Set up various robot components
+        self.robot = self.getSelf()  # Grab the robot reference from the supervisor to access various robot methods
+        self.positionSensorList = Func.get_All_positionSensors(self, self.timestep)
+        self.endEffector = self.getFromDef("endEffector")
+
+        # Select one of the targets
+        self.target = self.getFromDef("TARGET%s"%(np.random.randint(1, 10, 1)[0]))
+
+        self.setup_motors()
+
+        # Set up misc
+        self.stepsPerEpisode = 300  # How many steps to run each episode (changing this messes up the solved condition)
+        self.episodeScore = 0  # Score accumulated during an episode
+        self.episodeScoreList = []  # A list to save all the episode scores, used to check if task is solved
+        self.motorVelocity = 10
```

Comment:
Argument or constant?

Reply:
Similar to these [lines](https://github.com/aidudezzz/deepworlds/blob/5f86cdd69b4e9906b43ccc4acc3d31b3e60125b9/examples/cartpole/cartpole_discrete/controllers/robot_supervisor_manager/robot_supervisor.py#L77) in cartpole_discrete example.
I set these as constants, and users can modify these lines if they want.